smart-zeppelin/zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -224,7 +224,7 @@ public static void startZeppelinServer(ZeppelinConfiguration conf) throws Interr
     }
 
     jettyWebServer.join();
-    ZeppelinServer.notebook.getInterpreterSettingManager().close();
+    ZeppelinServer.notebook.getInterpreterSettingManager().close();*/
```

Comment:
This part blocks the thread in ZeppelinServer, but not sure whether it is proper to remove the code.